spot-contracts/contracts/RolloverVault.sol

```diff
@@ -79,9 +67,9 @@ contract RolloverVault is
 
     //-------------------------------------------------------------------------
     // Constants
-    uint8 public constant PERC_DECIMALS = 6;
-    uint256 public constant UNIT_PERC = 10**PERC_DECIMALS;
-    uint256 public constant HUNDRED_PERC = 100 * UNIT_PERC;
+    uint8 public constant PERC_DECIMALS = 8;
```

Comment:
I think we need a comment here that this is in the "multiplier space", not the "percent space". Are we consistent across the codebase?

Sorry, I've made a bunch on comments on this topic but I can see this causing a mistake at some point.

Reply:
Yea there are 2 consumers, perp and the vault. I renamed it to `FEE_POLICY_DECIMALS` and `ONE_FEE_PERC` which means 100% or 1.0

Reply:
I think that's clear 👍